ui/src/routes/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/sbom/index.tsx

```diff
@@ -0,0 +1,235 @@
+        Download SBOM reports from run {ortRun.index}
+      </h3>
+      <div className='grid grid-cols-2 gap-4'>
+        <Card>
+          <CardHeader>
+            <a
+              href='https://cyclonedx.org/'
+              target='_blank'
+              rel='noopener noreferrer'
+            >
+              <Tooltip>
+                <TooltipTrigger>
+                  <div className='flex flex-col items-center gap-4'>
+                    <img
+                      alt='CycloneDX'
+                      src={theme === 'dark' ? CycloneDXLight : CycloneDXDark}
+                      width={230}
+                    />
+                    <CardDescription className='text-left'>
+                      CycloneDX is a modern standard for the software supply
+                      chain. CycloneDX Bill of Materials (BOM) can represent
+                      full-stack inventory of software, hardware, services, and
+                      other types of assets. The specification originates and is
+                      led by the OWASP Foundation, standardized by Ecma
+                      International, and supported by the global information
+                      security community.
+                    </CardDescription>
+                  </div>
+                </TooltipTrigger>
+                <TooltipContent>Click to visit the homepage</TooltipContent>
+              </Tooltip>
+            </a>
+          </CardHeader>
+          <CardFooter className='flex justify-around gap-2'>
+            {cycloneDxReports && cycloneDxReports.length > 0
+              ? cycloneDxReports.map((filename) => (
+                  <div key={filename}>
+                    <Button
+                      variant='outline'
+                      className='h-auto whitespace-normal font-semibold text-blue-400'
+                      onClick={() => handleDownload(ortRun.id, filename)}
+                    >
+                      <div className='break-all'>
+                        {filename.includes('json')
+                          ? 'JSON format'
+                          : 'YAML format'}
+                      </div>
+                    </Button>
+                  </div>
+                ))
+              : 'No CycloneDX reports available.'}
+          </CardFooter>
+        </Card>
+        <Card>
+          <CardHeader>
+            <a
+              href='https://spdx.dev/'
+              target='_blank'
+              rel='noopener noreferrer'
+            >
+              <Tooltip>
+                <TooltipTrigger>
+                  <div className='flex flex-col items-center gap-4'>
+                    <img alt='SPDX' src={SPDX} width={150} />
+                    <CardDescription className='text-left'>
+                      System Package Data Exchange (SPDX®) is an open standard
+                      capable of representing systems with software components
+                      in as SBOMs (Software Bill of Materials) and other AI,
```

Comment:
Drop "in", and end the sentence after the closing parenthesis, omitting the rest. This should only be a short description, not a marketing punchline 😉